src/ucp/wireup/wireup.c

```diff
@@ -742,6 +785,25 @@ static void ucp_wireup_print_config(ucp_context_h context,
     }
 }
 
+static void ucp_wireup_print_lanes_config(ucp_ep_h ep, uint8_t *addr_indices)
+{
+    char str[32];
+
+    /* Print:
+     * - Detailed lanes config if log level is >= DEBUG
+     * - Less detailed info about lanes tls if log level is INFO
+     * - Nothing otherwise */
+
+    if (ucs_log_is_enabled(UCS_LOG_LEVEL_DEBUG)) {
+        snprintf(str, sizeof(str), "ep %p", ep);
+
+        ucp_wireup_print_config(ep->worker->context, &ucp_ep_config(ep)->key,
+                                str, addr_indices, UCS_LOG_LEVEL_DEBUG);
+    } else if (ucs_log_is_enabled(UCS_LOG_LEVEL_INFO)) {
+        ucp_wireup_print_used_tls(ep, UCP_NULL_RESOURCE);
```

Comment:
what is the reason to pass aux_rsc_index, if it's always NULL?

Reply:
no reason, need to remove